### PR TITLE
Add Secret Group

### PR DESCRIPTION
**Pull request by EngHabu**
# TL;DR
_Please replace this text with a description of what this PR accomplishes._

## Type
 - [ ] Bug Fix
 - [X] Feature
 - [ ] Plugin

## Tracking Issue
https://github.com/flyteorg/flyte/issues/800

### CODE REVIEW

File: protos/flyteidl/core/security.proto [hunk 1]
```diff
@@ -11,22 +11,28 @@ option go_package = "github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/core";
...
+        FILE = 2;
     }
 
     // The name of the secret to mount. This has to match an existing secret in the system. It's up to the implementation
     // of the secret management system to require case sensitivity.
-    string name = 1;
+    // +required
+    string key = 1;
+
+    // The name of the secret group where to find the key referenced above.
```
Comment (katrogan) [inline protos/flyteidl/core/security.proto:28]: add required or optional?

**Review by katrogan — DISMISSED** [review]
do you want to bump the versions in package.json and setup.py?

File: protos/flyteidl/core/security.proto (hunk 1 above)
Comment (EngHabu) [reply protos/flyteidl/core/security.proto:29]: done

**Comment by EngHabu** [comment]
done thanks for reminding me about bumping the versions... I always forget that

**Review by katrogan — APPROVED** [review]

**Pull request merged**